### DISCUSSION TITLE
1.1.1 업데이트

### DISCUSSION THREAD

**Discussion by x4cf** [naverz/zepeto-studio-kor#113] [4. General]
혹시 업데이트된 기능을 알 수 있을까요?
프로필 온오프 기능이랑 초대기능까지는 보았는데 다른 무엇이 또 업데이트 된것이 있을지 궁급합니다!

즐거운 업데이트 :D

**Comment by hyeonjin-jo** [comment]
안녕하세요

릴리즈 노트 페이지를 오늘 중으로 공개할 예정입니다

공개되면 한 번 더 댓글 남겨놓겠습니다

**Comment by hyeonjin-jo** [comment]
https://studio.zepeto.me/guides/world-release-note
릴리즈 노트 페이지 추가되었습니다.

감사합니다.
